.changeset/smooth-roses-act.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-query-builder': patch
+'@finos/legend-art': patch
+---
```

Comment:
Can you remove `@finos/legend-art` from the changeset as you have not made any changes to the package?